drivers/hmis/app/graphql/types/hmis_schema/enrollment_summary.rb

```diff
@@ -0,0 +1,45 @@
+module Types
+  class HmisSchema::EnrollmentSummary < Types::BaseObject
+    field :id, ID, null: false
+    field :entry_date, GraphQL::Types::ISO8601Date, null: false
+    field :project_name, String, null: false
+    field :project_type, Types::HmisSchema::Enums::ProjectType, null: false
+    field :project_id, String, null: false
+    field :move_in_date, GraphQL::Types::ISO8601Date, null: true
+    field :in_progress, Boolean, null: false
```

Comment:
```suggestion
    field :in_progress, Boolean, null: false, method: :in_progress?
```
I think this can replace the fn definition below